packages/api-client-core/src/operationRunners.ts

```diff
@@ -307,7 +322,7 @@ export const actionResultRunner = async <Action extends AnyActionFunction, Optio
         action.defaultSelection,
         response.data,
         backgroundAction.result,
-        action.modelSelectionField,
+        action.isBulk ? action.modelApiIdentifier : action.modelSelectionField,
```

Comment:
Unfortunate discrepancy, but makes sense since backgrounding a bulk action turns it into a set of individual actions.